### PR TITLE
Update Snowman Submission Page

### PR DESCRIPTION
**Pull request by yangashley**
Task: https://app.asana.com/0/1205655776549324/1207935835560266/f

### CODE REVIEW

File: intro-to-python/images/repl_invite.png
Comment (kelsey-steven-ada) [inline intro-to-python/images/repl_invite.png:1]: A couple thoughts on the images:
- Would it be possible to retake these and the other 2 existing screenshots with repl.it in dark mode? 
- It's a little hard for my eyes to tell, but if you aren't using [the Ada color palette](https://www.figma.com/design/gbldbUpIbUcxHG9ztZSAzg/Curriculum-Image-Colors?node-id=0-1&t=Nxwhh1roEi0wVpzi-1) fuschia for circles/bringing attention in images, we're trying to use that more consistently. 

If for time's sake we need to kick retaking the images, I can live with that, but we should make a ticket as a reminder to revisit.

File: intro-to-python/snowman.checkpoint.md [hunk 1]
```diff
@@ -25,29 +25,27 @@ You can also give the repl a new name here.
...
 
 ## Please Note
 
 The `repl` you've cloned has several files:
 
-1. `main.py` that will run the game. No need to modify this file.
-1. `game.py` that holds the important `snowman(snowman_words)` function we *will* modify
-1. `game_test.py` that holds a few written test that will allow us to test your code. No need to modify this file, but feel free to take a look at what these tests might expect.
+1. `main.py` that will run the game. You should not modify this file.
+1. `game.py` that holds the important `snowman(snowman_words)` function and you *will* modify
```
Comment (kelsey-steven-ada) [inline intro-to-python/snowman.checkpoint.md:39]:
```suggestion
1. `game.py` that holds the important `snowman(snowman_words)` function that you *will* modify
```

File: intro-to-python/snowman.checkpoint.md [hunk 2]
```diff
@@ -25,29 +25,27 @@ You can also give the repl a new name here.
...
 
 Several functions have already been made for you to *use* as shown below. You do not need to create them again.
 
-When adding your code, you only need to modify the `snowman(snowman_word)` function.  Note that this function takes a parameter `snowman_word`.  For testing purposes, we will be passing in the random word rather than using RandomWord.
+When adding your code, **you only need to modify the `snowman(snowman_word)` function**.  Note that this function takes a parameter `snowman_word`.  For testing purposes, we will be passing in the random word rather than using RandomWord.
 
 ---
 
-You should make use of the following functions from the previous lessons (already created for you):
+You should make use of the following functions from the previous lessons that are already created for you:
```
Comment (kelsey-steven-ada) [inline intro-to-python/snowman.checkpoint.md:48]:
```suggestion
You should make use of the following functions from the previous lessons which have been created for you:
```

File: intro-to-python/images/repl_invite.png
Comment (yangashley) [reply intro-to-python/images/repl_invite.png:1]: Yes can do that! I didn't even realize replit had a dark mode. I have my system in dark mode, but replit was still white so thanks for calling that out.

**Review by kelsey-steven-ada — APPROVED** [review]

File: intro-to-python/images/repl_invite.png
Comment (yangashley) [reply intro-to-python/images/repl_invite.png:1]: Yeah I read that note and then proceeded to forget to do that. I'll get em pushed up soon!

**Pull request merged**